eventhubs/client/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 5.0.0-preview.1 (2019-07-01)
+For release notes and more information please visit https://aka.ms/azure-sdk-preview1-java
```

Comment:
nit: everywhere (in Key Vault and python sdk) else the link is in the next line.